tests/data_files/Readme-x509.txt

```diff
@@ -91,3 +95,13 @@ Signing CA in parentheses (same meaning as certificates).
 
 Note: crl_future would revoke server9 and cert_sha384.crt if signed by CA 1
       crl-rsa-pss* would revoke server6.crt if signed by CA 2
+
+Generation
+----------
+
+Newer test files have been generated through commands in the Makefile. The
+resulting files are committed to the repository so that the tests can
+run without having to re-do the generation and so that the output is the
+same for everyone (the generation process is randomized).
```

Comment:
Big +1 on that. Test certificates are a pain to maintain, and while I've tried to improve documentation, I think experience proves it's too easy to forget, so I think executable documentation is a better way to go.